.werft/eks-installer-tests.yaml

```diff
@@ -24,6 +24,10 @@ args:
   desc: "Set this to true to run KOTS upgrade from the specified version to the latest version"
```

Comment:
I am wondering if we should make this changes also to `k3s-installer-tests.yaml` and `aks-installer-tests.yaml`.

Reply:
Ah, Added!